app/src/main/java/db/dto/CardStackDTO.kt

```diff
@@ -0,0 +1,19 @@
+package db.dto
+
+import db.entities.CardStack
+import java.io.Serializable
+
+data class CardStackDTO(
+    val active: Boolean,
+    val cards: List<CardDTO>,
+    val created_at: String,
+    val created_by: String,
+    val id: Long,
+    val name: String,
+    val num_cards: Int,
```

Comment:
you should use camelCase names in your code and annotate json fields to indicate fields that are expected in json. See https://sites.google.com/site/gson/gson-user-guide#TOC-Custom-Serialization-and-Deserialization for reference. It's not a good style to mix camel case and snake case together. By annotating fields you also make your code more robust as you will have to adjust json field name only in one place should it change in the future without breaking your code.